ydb/core/persqueue/pq_impl.cpp

```diff
@@ -3031,6 +3309,7 @@ void TPersQueue::EndWriteTxs(const NKikimrClient::TResponse& resp,
 
     SendReplies(ctx);
     CheckChangedTxStates(ctx);
+    ForwardGetOwnershipToShadowPartitions(ctx);
```

Comment:
Change this func to CreateSupportivePartitions

Reply:
done
https://github.com/ydb-platform/ydb/pull/1037/commits/74c832f44de40a1c11abdeb95e1b95dd7266a9bc